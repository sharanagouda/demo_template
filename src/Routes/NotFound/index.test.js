import { NotFound } from "./index";

jest.mock("@discovery-dni/cd-tv.platform", () => ({
  androidKeyboardVisible$: jest.fn()
}));

test("returns correct data", () => {
  const component = new NotFound().render();

  expect(component).toMatchSnapshot();
});
