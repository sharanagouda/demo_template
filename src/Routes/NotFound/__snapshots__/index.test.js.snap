// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns correct data 1`] = `
<div>
  <h1 />
  <pre />
</div>
`;
